src/tests/createHeadlessForm.test.js

```diff
@@ -780,6 +781,29 @@ describe('createHeadlessForm', () => {
       expect(() => fieldValidator.validateSync('')).toThrowError('The value must be a number');
     });
 
+    it('shows an error when positive number is entered into a "number" field with maximum set to zero', () => {
```

Comment:
suggestion: First of all, we know that this file deserves some love as it's huge and a little messy. But could I ask to move this test to inside the `describe('when a field is number'` (line ~1812) so it's closer to the other ones related to min/max assertions?

Regarding validating the field itself, the way you did is the original way we did it, but now we use `handleValidation` directly to be closer to what the tool actually does. You can check "'applies correct validation for fieldset fields'" as an example. For you case, it would look like this:

```
const { handleValidation } = createHeadlessForm(schemaInputTypeNumberZeroMaximum);
const validateForm = (vals) => friendlyError(handleValidation(vals));

expect(validateForm({ tabs: 1 })).toEqual({
  tabs: 'Must be smaller or equal to 0',
});

```